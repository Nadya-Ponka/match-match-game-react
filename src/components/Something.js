import React from 'react';

class Something extends React.Component {
    render() {
        return (
            <p>This game was developed using the React and the Redux.
            There are two middleware-functions: for console.log common App-state
            after each action and fetching data from webAPI.
            CombineReducers was used for logical splitting reducer-functions.
            Lines in LeaderTable were limited by count 55 for the verification period
            to provide seeing your result.
            React Router is using at the win-screen.
            Three components have snapshotTest including one component with connect()
            and two reducer-functions have test too.</p>
        );
    }
}

export default Something;
