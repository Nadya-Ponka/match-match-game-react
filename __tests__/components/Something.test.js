import React from 'react';
import { shallow } from 'enzyme';

import Something from '../../src/components/Something';

describe('Something', () => {
    test('match snapshot', () => {
        const wrapper = shallow(<Something />);
        expect(wrapper).toMatchSnapshot();
    });
});
