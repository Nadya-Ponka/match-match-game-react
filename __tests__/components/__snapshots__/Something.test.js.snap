// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Something match snapshot 1`] = `
<p>
  This game was developed using the React and the Redux. There are two middleware-functions: for console.log common App-state after each action and fetching data from webAPI. CombineReducers was used for logical splitting reducer-functions. Lines in LeaderTable were limited by count 55 for the verification period to provide seeing your result. React Router is using at the win-screen. Three components have snapshotTest including one component with connect() and two reducer-functions have test too.
</p>
`;
